### Laelaps laelaps_control/ReadGpio service
#
# Read GPIO pin value(s)
#

uint8[] pin     # pin states (s) to read
---
Gpio[] gpio     # resulting pin state(s)
